projects/umo/config/settings.yml

```diff
@@ -0,0 +1,19 @@
+# Beware! This file is public by definition
+default: &default
+  default_team_name: 'Demo-team'
```

Comment:
should this be UMO? or does it not matter?

Reply:
We can change it once we make a team. It should correspond to an existing team name in seeds/teams. It's for when we use OTR responses, to add those anonymously created people to a team/organization.